spec/index.html

```diff
@@ -10424,7 +10424,205 @@ <h3>Evaluation Semantics</h3>
+            </p>
+            </div>
+            <section>
+              <h4>Syntax Restriction</h4>
+              <p><i>Additional [[[#sparqlGrammar]]] note:</i></p>
+                <blockquote>
+                Any variable that is assigned to in the graph pattern of `EXISTS`/`NOT EXISTS` must not be in-scope.
+                This applies to `BIND`, variables introduced by `AS` in a `SELECT` clause, variables in a `VALUES`
+                clause, and variables introduced by `AS` in `GROUP BY`.
+              </blockquote>
+              <p>
+                Extend the "in-scope" rules to include the possible 
+                variables that are in-scope for the current row:
+              </p>
+              <table style="border-collapse: collapse; border-color: #000000; border-spacing:5px; border-width: 1px">
+                <tbody>
+                  <tr>
+                    <th>Syntax Form</th>
+                    <th>In-scope variables</th>
+                  </tr>
+                  <tr>
+                    <td>`EXISTS` and `NOT EXISTS` filters </td>
+                    <td><code>v</code> is in-scope if it is in-scope for the pattern to which the `FILTER` is applied.
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+              <div class="note">
+                <p>
+                  This restriction means that <a href="#defn_valuesinsertion">values inserted</a>
+                  do not conflict with values assigned to variables within the pattern.
+                </p>
+                <p>
+                  This operation is performed as part of query parsing.
+                </p>
+              </div>
+            </section>
+            <section>
+              <h4>Remapping</h4>
+              <p>
+                Remapping ensures that a variable name used inside a project expression,
+                but which is not part of the results of the evaluation of the project expression,
+                does not coincide with a variable mentioned anywhere else in the
+                algebra expression of a query.
+              </p>
+              <p>
+                Renaming these variables does not change the results of evaluating
+                the project expresssion.
+              </p>
+              <div class="defn">
+                <b>Definition: <span id="defn_projmap" name="defn_projmap">Projection Expression Variable Remapping</span></b>
+                <p>
+                  For a projection algebra operation #sparqlProjection `Project(A, PV)` acting on algreg express `A` and with set of variables `PV`, define
+                  a partial mapping `F` from
+                  `<a href="#sparqlQueryVariables">V</a>`,
+                  the set of all variables, to `V` where:
+                </p>
+                <pre>F(v) = v1 if v is in PV, where v1 is a fresh variable
+F(v) = v if v is not in PV</pre>
```

Comment:
Considering the second line of this definition, it seems to me that `F` is actually not partial.

If I am wrong with this assumption, can you explain which variables are mapped by `F` and which ones are not.

Reply:
No longer partial.